content/docs/2.7/concepts/scaling-deployments.md

```diff
@@ -155,10 +153,9 @@ Due to the HPA metric being of type `AverageValue` (see below), this will have t
 **Example:** When my instance of prometheus is unavailable 3 consecutive times, KEDA will change the HPA metric such that the deployment will scale to 6 replicas.
 
 There are a few limitations to using a fallback:
- - It is **not** supported by the CPU & memory scalers and will assume that fallback is disabled in those cases. This is because it only supports scalers that present their target as an `AverageValue` metric.
+ - It only supports scalers if their target is an `AverageValue` metric. Thus, it is **not** supported by the CPU & memory scalers, or by scalers whose metric target type is `Value`, and will assume that fallback is disabled in those cases.
```

Comment:
one idea to make this slightly clearer:

```suggestion
 - It only supports scalers whose target is an `AverageValue` metric. Thus, it is **not** supported by the CPU & memory scalers, or by scalers whose metric target type is `Value`. In these cases, it will assume that fallback is disabled.
```